docs/team/transfer-apps.md

```diff
@@ -0,0 +1,30 @@
+title: Sematext Transfer Apps
+description: Inviting team members to your account means they get access to all your Apps, dashboards, notification hooks, alert rules, and everything else!
+
+No level of Access can change the Owner status of an App. To transfer the 
+ownership of an App, use the Transfer Apps feature. Click the **Start Ownership Transfer** 
+button to open the side modal, select an app, click **Transfer to...** to proceed, enter the email of the
+user you want to transfer the App to, and hit **Transfer Ownership**.
```

Comment:
Add:
Note that Apps can be transferred only to existing Sematext users.
(assuming this is correct)

Reply:
@FexCore this?

Reply:
This is fixed.